pulsar-broker/src/main/java/org/apache/pulsar/broker/web/PulsarWebResource.java

```diff
@@ -924,7 +924,7 @@ public void validatePoliciesReadOnlyAccess() {
                 throw new RestException(Status.FORBIDDEN, "Broker is forbidden to do read-write operations");
             }
         } catch (Exception e) {
-            log.warn("Unable to fetch read-only policy config {}", e);
+            log.warn("Unable to fetch read-only policy config ", e);
```

Comment:
In slf4j you don't need to pass a placeholder for exceptions

Reply:
Yes, I don't paas a placeholder, I am deleteing it

Reply:
Oh sorry. I need a coffee.
I misread the diff

Reply:
Though the behavior with or without placeholder is different, and in some cases we really mean to have the placeholder there. eg: 

 * With placeholder: only the exception message is printed
 * Without placeholder: the full stacks trace is printed

For common, expected errors we don't want to pollute the logs with lots of stack traces that are not very useful.